common.blocks/select/select.bemhtml

```diff
@@ -21,7 +21,7 @@ block('select')(
                         iterateOptions(option.group);
                     } else {
                         firstOption || (firstOption = option);
-                        optionIds.push(option.id = _this.generateId());
+                        optionIds.push(option.id = 'option' + random++);
```

Comment:
as I can see in [code](https://github.com/bem/bh/blob/dff393fa5c6c7f1d0a327a1c2f43895bcd23a0c7/lib/bh.js#L256) and [specs](https://github.com/bem/bh/blob/dff393fa5c6c7f1d0a327a1c2f43895bcd23a0c7/test/test.generateId.js#L10) `generateId` in BH works different (also incorrect) and generate different ids every time — so we can just do `option.id = ctx.generateId()`